site/en/gemini-api/docs/prompting_with_media.ipynb

```diff
@@ -387,7 +386,29 @@
         "\n",
         "You can use video data for prompting with the `gemini-1.5-pro` model. However, video file formats are not supported as direct inputs by the Gemini API. You can use video data as prompt input by breaking down the video into a series of still frame images and a separate audio file. This approach lets you manage the amount of data, and the level of detail provided by the video, by choosing how many frames per second are included in your prompt from the video file.\n",
         "\n",
-        "Note: Video files added to a prompt as constituent parts, audio file and image frames, are considered as separate prompt data inputs by the model. For this reason, requests or questions that specify the time when *both* an audio snippet and video frames appear in the source video may not produce useful results."
+        "Note: Video files added to a prompt as constituent parts, audio file and image frames, are considered as separate prompt data inputs by the model. For this reason, requests or questions that specify the time when *both* an audio snippet and video frames appear in the source video may not produce useful results.\n",
+        "\n",
+        "### Plain text formats\n",
+        "\n",
+        "The File API supports uploading plain text files with the following MIME types:\n",
+        "-   text/plain\n",
+        "-   text/html \n",
+        "-   text/css\n",
+        "-   text/javascript\n",
+        "-   application/x-javascript\n",
+        "-   text/x-typescript\n",
+        "-   application/x-typescript\n",
+        "-   text/csv\n",
+        "-   text/markdown\n",
+        "-   text/x-python\n",
+        "-   application/x-python-code\n",
+        "-   application/json\n",
+        "-   text/xml\n",
+        "-   application/rtf\n",
+        "-   text/rtf\n",
+        "\n",
+        "For plain text files with a MIME type not on the list, you can try specifying\n",
+        "the MIME type manually."
```

Comment:
```suggestion
        "one of the above MIME types manually."
```

Nit: To differentiate between "the MIME type" (which may not be supported) and one that will, like text/plain.